config.js

```diff
@@ -60,6 +60,18 @@ module.exports = {
 
     // The number of transactions we buffer before we publish to the Cloud Trace
     // API, unless we hit `flushDelaySeconds` first.
-    bufferSize: 1000
+    bufferSize: 1000,
+
+    // If true, the trace agent will attempt to publish any buffered traces if an
+    // uncaught exception occurs. This will result in the error being silenced. If
+    // you want your application to exit on uncaught exception you will have to
+    // add your own listener to the 'uncaughtException' event on the process.
+    publishOnUncaught: false,
```

Comment:
Another idea would be to expose a single option `onUncaughtExcpetion` that has the following behaviours:
* falsy: we don't deal with uncaught exceptions
* 'flush': flush the buffer, and silence the error
* 'flushAndExit': flushes then exits.

WDYT?